mlflow/tracking/fluent.py

```diff
@@ -284,6 +298,13 @@ def start_run(
         exp_id_for_run = experiment_id if experiment_id is not None else _get_experiment_id()
 
         user_specified_tags = deepcopy(tags) or {}
+        if description:
+            if MLFLOW_RUN_NOTE in user_specified_tags:
+                raise MlflowException(
+                    f"Description is already set via the tag {MLFLOW_RUN_NOTE} in tags."
+                    f"Remove the key {MLFLOW_RUN_NOTE} from the tags or omit the description."
```

Comment:
Nit: Can we use error code `INVALID_PARAMETER_VALUE` here?

Reply:
Sure, I think I've added this now.